GTG/gtk/data/modify_tags.ui

```diff
@@ -20,7 +20,7 @@
             <property name="layout_style">end</property>
             <child>
               <object class="GtkButton" id="modifytags_ok">
-                <property name="label" translatable="yes">Add/Remove Tags</property>
+                <property name="label" translatable="yes">Apply</property>
```

Comment:
Describing the action to be done is according to the HIG, while a generic "OK" or "Apply" isn't; though as the original was a confusing "Add/Remove Tags" it's probably fine/better, but it could be something even better; e.g. "Save tag modifications" or something, but I didn't go looking for the context of this at all.